airflow/contrib/hooks/gcp_compute_hook.py

```diff
@@ -21,9 +21,14 @@
 import warnings
 
 # pylint: disable=unused-import
-from airflow.gcp.hooks.compute import GceHook, GceOperationStatus  # noqa
+from airflow.gcp.hooks.compute import ComputeEngineHook, GceOperationStatus  # noqa
 
-warnings.warn(
-    "This module is deprecated. Please use `airflow.gcp.hooks.compute`.",
-    DeprecationWarning, stacklevel=2
-)
+
+class GceHook(ComputeEngineHook):
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "This module is deprecated. Please use `airflow.gcp.hooks.compute`.",
```

Comment:
The message should include also information about the new operator that should be used by the user.
```suggestion
            "This module and operator are deprecated. Please use `airflow.gcp.hooks.compute.ComputeEngineOperator`.",
```